### DISCUSSION TITLE
组件是否启用，可以考虑使用一个标识来决定，否则很多默认配置需要运行时赋予

### DISCUSSION THREAD

**Discussion by visiky** [antvis/G2Plot#2729] [Ideas]
![image](https://user-images.githubusercontent.com/15646325/126920558-85f9f237-a550-4ba5-bb3a-59365ff9f306.png)

**Comment by visiky** [comment]
类似这样的默认配置项无法设置在 default Options 中，因为 label 默认不展示，所以需要置空

**Reply by hustcc** [reply]
和 interaction 一样，要关闭 g2 默认内置开启的吧？

**Reply by visiky** [reply]
是 G2Plot 自己内置了，但是用户默认又不知道；无法通过 `Plot.getDefaultOptions()` 感知

**Reply by hustcc** [reply]
那就增加吧~
